public/index.pug

```diff
@@ -52,4 +50,10 @@ html(lang="en")
             li
               a(ui-sref="login") Login            
     ui-view
- 
+    div.page-footer.navbar-default
+      .container
+        div(style="display: inline")
+          .footer-copyright(style="float: left") © 2018 Stanford University. All rights reserved. 
+            a(ui-sref="about") Terms of use.
+          .footer(style="float: right") ISBN XXXX DOI XXXX OCLC XXX Published by Center for Spatial and Textual Analysis (CESTA) Stanford University
```

Comment:
Footer is not a "sticky" footer since it isn't always at the bottom of the page ... we could fix later though.


![image](https://user-images.githubusercontent.com/1689183/66275887-1fc4fd80-e842-11e9-9672-005a03479ba3.png)